hack/ci/testdata/kubermatic.yaml

```diff
@@ -31,6 +31,7 @@ spec:
     # VPA won't do anything useful due to missing Prometheus, but we can
     # at least ensure we deploy a working set of Deployments.
     VerticalPodAutoscaler: true
+    OperatingSystemManager: true
```

Comment:
This would enable the feature flag for all e2e tests. What effects will this have?